src/components/charts/bubble-chart/bubble-chart-styles.scss

```diff
@@ -1,7 +1,29 @@
+@import '~styles/settings.scss';
+
 .circle {
   cursor: pointer;
 }
 
-.tooltip span {
-  text-align: left !important;
+:global {
```

Comment:
We prefer to use scoped classes than global. so, in this case, you can just skip the global and import it as styles.bubbleChartTooltip. We are using styled components: https://github.com/styled-components/styled-components

Reply:
Yes, sadly this is one of two ways where overwriting default styles of ReactTooltip component works. In the [documentation of react-tooltip](https://www.npmjs.com/package/react-tooltip), they are saying that we should use `!important` to overwrite some default css rules:
![screenshot from 2018-11-13 12-18-10](https://user-images.githubusercontent.com/15097138/48413146-bc6c4180-e73e-11e8-812a-9a9c7f2e98cf.png)
I spend some time to find a more elegant solution but I didn't' find anything yet.
I can remove `:global` and change it to `!important` where it is needed, but both don't look good :(

Reply:
Mmm, yes, actually what we do normally is to create a tooltip component but as its only the styles that we are changing we may leave it like this

Reply:
So I left it with :global